src/other/sample.jl

```diff
@@ -0,0 +1,29 @@
+import StatsBase: sample
+
+function sample(df::AbstractDataFrame; replace::Bool=true, ordered::Bool=false)
```

Comment:
Just do `N::Integer=1` in the definition below, and you'll get this one for free.